Softeq.XToolkit.WhiteLabel.Tests/ViewModelStub.cs

```diff
@@ -11,4 +12,9 @@ public class ViewModelStub : ViewModelBase
         public string StringParameter { get; set; }
         public object ObjectParameter { get; set; }
     }
+
+    public class ViewModelStub<T> : ViewModelBase, IViewModelParameter<T>
```

Comment:
As I can see `IViewModelParameter` is obsolete and currently not used. I think we need to remove this interface.

Reply:
Done